apps/palette-tauri/src/styles.css

```
@@ -36,20 +82,86 @@ textarea {
+  padding-top: max(0px, calc((100vh - 438px) * 0.29));
+}
+
+.tauri-runtime #app {
+  place-items: stretch;
+  padding-top: 0;
 }
 
 .palette-shell {
+  --ctl: 38px;
+  --r-ctl: 10px;
   display: flex;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)